tst/com/vivian/java/leetcode/CCI4_3ArrayToBSTTest.java

```diff
@@ -0,0 +1,38 @@
+package com.vivian.java.leetcode;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CCI4_3ArrayToBSTTest {
+    CCI4_3ArrayToBST solution = new CCI4_3ArrayToBST();
+    BiTreeNode<Integer>n1 = new BiTreeNode<>(1, null, null);
+    BiTreeNode<Integer>n5 = new BiTreeNode<>(5, null, null);
+    BiTreeNode<Integer>n8 = new BiTreeNode<>(8, null, null);
+     BiTreeNode<Integer>n3 = new BiTreeNode<>(3, null, null);
+     BiTreeNode<Integer>n7 = new BiTreeNode<>(7, null, n8);
+     BiTreeNode<Integer>n6 = new BiTreeNode<>(6, n5, n7);
+    BiTreeNode<Integer>n2 = new BiTreeNode<>(2, n1, n3);
+    BiTreeNode<Integer>n4 = new BiTreeNode<>(4, n2, n6);
+    
+    
+
+    int[] arr = new int [] {1, 2, 3, 4, 5, 6, 7, 8};
+
+    public boolean sameTree(BiTreeNode<Integer> root1, BiTreeNode<Integer> root2) {
```

Comment:
The naming suggest you are trying to verify if root1 and root2 reference to the same object or not, which doesn't match the exact behavior of this method. 

You should rename it to ```treeEquals```.

Reply:
updated